px-data-grid.html

```diff
@@ -637,6 +637,14 @@
               observer: '_gridHeightChanged'
             },
 
+            /* Sets the px-grid to a mode where it will follow the parent
+            container height. Demo on /demo/px-data-grid-height-demo.html as it doesn't
```

Comment:
No need in the second sentence as well as in the demo page which doesn't fit the px-demo component.

After we merge this PR, we will make jsfiddle with working example and attach it to the reported issue.